ldm/invoke/concepts_lib.py

```diff
@@ -22,8 +23,8 @@ def __init__(self, root=None):
         self.concepts_loaded = dict()
         self.triggers = dict()            # concept name to trigger phrase
         self.concept_names = dict()       # trigger phrase to concept name
-        self.match_trigger = re.compile('(<[\w\-]+>)')
-        self.match_concept = re.compile('<([\w\-]+)>')
+        self.match_trigger = re.compile('(<[^>]+>)') # trigger is less restrictive than HF concept name
```

Comment:
i am not sure how else to match, because they seem unbound. the issue came up because <3d-cyborg-female> turns into <A cyborg female>. i guess the regex could be `(<[\w\- >]+>)` (just add support for matching ' '). actually yeah this would be better. i'll push a new commit.

Reply:
we really don't know what trigger strings people are going to use, like maybe there's an embedding with the trigger `🐈`, or some kind of ümläüted string. i guess this will need to be fixed on a case-by-case basis